doc/rst/platforms/cray.rst

```diff
@@ -82,27 +99,28 @@ installed, do the following:
 
 
 Currently the number of Chapel configurations available on
-HPE Cray EX systems is quite limited.  Only the following have been built
+HPE Cray EX systems is somewhat limited.  Only the following have been built
```

Comment:
Noticed when looking at a different documentation change, but I think this should be "HPE Cray EX systems *are* somewhat limited"

Reply:
Not that I'm an English major, but no, I think the verb needs to match up with "number", which is singular. As a test, see how it scans with the prepositional phrase removed: "the number is limited" vs. "the number are limited".

Reply:
D'oh, that's what happens when I respond to a single line instead of the whole sentence.  Sorry to bother you!

Reply:
No probs -- in other circumstances that same thing would be valued, as: "has the ability to concentrate on the work at hand despite distractions."   :-)